python/Dockerfile

```diff
@@ -21,6 +21,8 @@ RUN sed -i -- 's/\"disable_updater\": false/\"disable_updater\": true/g' /google
 RUN mkdir /.ssh
 ENV PATH /google-cloud-sdk/bin:$PATH
 ADD ./ /root/python
+RUN pip install virtualenv virtualenvwrapper
+RUN bash /root/python/venv/bin/activate
```

Comment:
source didn't exist on these machines, and is basically just executes bash anyway in this case.